tutorial/csharp.async/README.md

```diff
@@ -0,0 +1,8 @@
+# Tutorial Frameword 4.5+ Clent Async
+
+## Setp
+1. thrift -gen csharp:async shared.thrift
+2. thrift -r -gen csharp:async tutorial.thrift
```

Comment:
thrift -r generates the file AND all included files. Is step 1 really necessary then?